rom/dev/tests/rom_integration_tests/test_fmcalias_derivation.rs

```diff
@@ -573,12 +578,11 @@ fn test_pcr_log_across_update_reset() {
 
 #[test]
 fn test_fuse_log() {
-    const FMC_SVN: u32 = 4;
+    const FW_SVN: u32 = 4;
+    const FW_SVN_FUSE: u32 = 3;
 
     let fuses = Fuses {
-        anti_rollback_disable: true,
-        fmc_key_manifest_svn: 0x0F,  // Value of FMC_SVN
-        runtime_svn: [0xF, 0, 0, 0], // Value of RT_SVN
+        fw_svn: [0x7, 0, 0, 0], // Value of FW_SVN_FUSE
```

Comment:
Same, would be helpful to have code convert this so it's more obvious where this value comes from.

Reply:
Agreed. Will add in a follow-up.